freqtrade/optimize/optimize_reports/bt_output.py

```diff
@@ -132,8 +132,13 @@ def text_table_periodic_breakdown(
     :param days_breakdown_stats: Days breakdown metrics
     :param stake_currency: Stakecurrency used
     """
+    if period == "weekday":
+        first_column = "Week day"
```

Comment:
Is this (whole part) actually necessary? 
I think "Weekday" (which would be the default formatting) is just as correct (based on some sites on the internet, more commonly used) as "Week day".

Funnily - that's also confirmed by your screenshot - which uses weekday, too.

Personally, i'd revert the change to this file entirely - as it's not actually necessary.